src/CONST.ts

```diff
@@ -1429,6 +1436,7 @@ const CONST = {
             DELETE: 'delete',
             APPROVE: 'approve',
             TRACK: 'track',
+            MOVE: 'move',
```

Comment:
Why was this constant added? I can't find it referenced anywhere so I am thinking of removing it in [this PR](https://github.com/Expensify/App/pull/40358).

Reply:
OK, thank you! I'll move forward with removing it.